app/parse.py

```
@@ -13,11 +39,197 @@ class Product:
     price: float
     rating: int
     num_of_reviews: int
+    additional_info: dict
+
+
+PRODUCT_FIELDS = [f.name for f in fields(Product)]
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)8s]:  %(message)s",
+    handlers=[
+        logging.FileHandler("products.log"),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+
+# === HELPERS ===
+def accept_cookies_if_present(driver: WebDriver) -> None:
+    """Click 'Accept cookies' if the banner appears."""
+    try:
+        WebDriverWait(driver, 3).until(
+            EC.presence_of_element_located((By.ID, "accept-cookies"))
+        )
+        driver.find_element(By.ID, "accept-cookies").click()
+    except Exception:
+        pass
```

This `except Exception:` is very broad. It can silently hide unexpected errors unrelated to the cookie banner check. Prefer catching specific exceptions, e.g. `TimeoutException`, `NoSuchElementException`, or handle `Exception` but log a debug message so real problems are visible during development.